packages/opencensus-core/test/test-span.ts

```diff
@@ -258,4 +258,35 @@ describe('Span', () => {
       assert.ok(instanceOfLink(span.messageEvents[0]));
     });
   });
+
+  /**
+   * Should set a status
+   */
+  describe('addStatus()', () => {
+    it('should return default status', () => {
+      const rootSpan = new RootSpan(tracer);
+      rootSpan.start();
+
+      const span = new Span(rootSpan);
+      span.start();
+
+      assert.equal(rootSpan.status.code, 0);
+      assert.equal(rootSpan.status.message, null);
+      assert.equal(span.status.code, 0);
+      assert.equal(span.status.message, null);
+    });
+
+    it('should set an error status', () => {
+      const rootSpan = new RootSpan(tracer);
+      rootSpan.start();
+      const span = new Span(rootSpan);
+      span.start();
+      span.setStatus(400, 'This is an error');
```

Comment:
Per comment about about the canonical values, could this be a number like  `span.SetSTatus('3', 'Invalid argument')` ?